libraries/WebServer/examples/WebServer/README.md

```diff
@@ -6,9 +6,9 @@ It is a small project in it's own and has some files to use on the web server to
 
 This example requires some space for a filesystem and runs fine boards with 4 MByte flash using the following options:
```

Comment:
Please update also this line as suggested, so its clear that the next part is about partition tables.
```suggestion
This example requires some space for a filesystem and runs fine on supported SoCs with 4 MByte or more flash by selecting proper partition table:
```

Reply:
yes, better wording.Thanks